scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowDoFns.scala

```diff
@@ -51,16 +49,16 @@ private[this] abstract class PredictDoFn[T, V, M <: Model[_]](
   def processElement(c: DoFn[T, V]#ProcessContext): Unit = {
     val result = withRunner { runner =>
       val input = c.element()
-      val i = inFn(input)
+      val i = extractInput(input)
       var result: V = null.asInstanceOf[V]
 
       try {
         i.foreach { case (op, t) => runner.feed(op, t) }
-        fetchOp.foreach(runner.fetch)
+        outputTensorNames.foreach(runner.fetch)
```

Comment:
some comments made on #2342 apply here

Reply:
rigth in this case here is not needed; the thing is that the other PR was doing extra processing on the input; 👍